contract_function_result.go

```diff
@@ -108,14 +108,6 @@ func _ContractFunctionResultFromProtobuf(pb *services.ContractFunctionResult) Co
 		infos[i] = _ContractLogInfoFromProtobuf(info)
 	}
 
-	createdContractIDs := make([]ContractID, 0)
```

Comment:
A bit confused as to why this is being removed.

Reply:
Because lint was complaining about using deprecated variable.